src/portable/renesas/usba/dcd_usba.c

```diff
@@ -513,7 +566,13 @@ void dcd_init(uint8_t rhport)
   USB0.SYSCFG.BIT.DCFM = 0;
   USB0.SYSCFG.BIT.USBE = 1;
 
+  USB0.PHYSLEW.LONG = 0x5;
```

Comment:
The `PHYSLEW` register is not present on RX63N. It should be wrapped with ifdef.